### ISSUE TITLE
Blog Post date always shows current month, not published date's month

### ISSUE THREAD

**Issue by guategeek** [getgrav/grav-theme-receptar#16]
Blog Posts on the Receptar theme always display as being published on the current month, the rest of the date (day and year) are correct but the month is not.

I don't know twig so can't figure out how to fix this.

**Comment by kvandenbrande** [comment]
Same issue, solved it by editing /user/themes/receptar/templates/partials/blog_item.html.twig 
around line 40 change:
`<div class="entry-meta entry-meta-bottom">
        <time datetime="{{ 'MONTHS_OF_THE_YEAR'|ta(post.date|date('n') - 1) }} {{ page.date|date("d, Y") }}" class="entry-date entry-meta-element published" title="{{ 'MONTHS_OF_THE_YEAR'|ta(post.date|date('n') - 1) }} {{ page.date|date("d, Y") }}" itemprop="datePublished">{{ 'MONTHS_OF_THE_YEAR'|ta(post.date|date('n') - 1) }} {{ page.date|date("d, Y") }}</time>`

to:
`      <div class="entry-meta entry-meta-bottom">
        <time 	       datetime=" {{ page.date|date("M d Y") }}" 
	 class="entry-date entry-meta-element published" 
			  title=" {{ page.date|date("M d Y") }}" 
	itemprop="datePublished"> {{ page.date|date("M d Y") }}
	</time>` 
Depending how you like time format.

**Comment by jonata** [comment]
Yes, I had this problem too. So the problem is that we were using `blog.date` instead of `page.date`. But probably `blog` should pass the correct date anyway.
